qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/LocationManagementSteps.java

```diff
@@ -0,0 +1,157 @@
+
+public class LocationManagementSteps extends Steps {
+    private static final String TAG_NAME = "This is a tag name";
+    private static final String MAX_VALUE = "20";
+    private static final String MIN_VALUE = "1";
+    private static final String DESCRIPTION = "This is the description";
+    private static final String NAME = "Noah";
+    private static final String LOCATION_NAME = "Bugema";
+    private static final String ADDRESS_ONE = "Bugema one";
+    private static final String ADDRESS_TWO = "Bugema two";
+    private static final String CITY = "Kampala";
+    private static final String STATE = "Central";
+    private static final String COUNTRY = "Uganda";
+    
+    private ConfigureMetadataPage configureMetadataPage;
+    private LocationPage locationPage;
+    private ManageLocationsPage manageLocationsPage;
+    private AddEditLocationPage addEditLocationPage;
+    private ManageLocationTagsPage manageLocationTagsPage;
+    private ManageLocationAttributeTypesPage manageLocationAttributeTypesPage;
+
+    @Before(RunTest.HOOK.SELENIUM_LOCATION_MANAGEMENT)
+    public void visitHomePage() {
+        initiateWithLogin();
+    }
+
+    @After(RunTest.HOOK.SELENIUM_LOCATION_MANAGEMENT)
+    public void destroy() {
+        quit();
+    }
+
+    @Given("a user clicks on the configure metadata link from the home page")
+    public void clickOnConfigureMetaDataLink(){
+        configureMetadataPage = homePage.goToConfigureMetadata();
+    }
+
+    @When("the system loads configure metadata link dashboard")
+    public void loadsConfigureMetaData(){
+        assertTrue(textExists("Configure Metadata"));
+    }
+
+    @And("a user clicks on Manage Location Attribute Types")
+    public void clickOnManageLocationAttributesTypes(){
+        manageLocationAttributeTypesPage = configureMetadataPage.goToManageLocationAttributeTypesPage();
+    }
+
+    @And("a user clicks on add new location attribute type")
+    public void addNewLocationAttributeType(){
+        manageLocationAttributeTypesPage.goToAddNewLocationAttributeTypeForm();
+    }
+
+    @And("a user fills the form")
+    public void fillsTheForm(){
+        manageLocationAttributeTypesPage.enterLocationAttributeTypeName(NAME);
+        manageLocationAttributeTypesPage.enterLocationAttributeTypeDescription(DESCRIPTION);
+        manageLocationAttributeTypesPage.enterMinimumOccurs(MIN_VALUE);
+        manageLocationAttributeTypesPage.enterMaximumOccurs(MAX_VALUE);
+        manageLocationAttributeTypesPage.selectDatatype("Provider Datatype");
+        manageLocationAttributeTypesPage.enterDatatypeConfiguration("I have entered the data configuration");
+        manageLocationAttributeTypesPage.selectPreferredHandler("Provider Field Gen Datatype Handler");
+        manageLocationAttributeTypesPage.enterHandlerConfiguration("I have entered the handler configuration");
+    }
+
+    @Then("the user saves the form")
+    public void userSavesForm(){
```

Comment:
Please this should have space. i
```suggestion
    public void userSavesForm() {
```Ensure all your method definition classes have space like the suggested above.